documentation/crds_users_guide/source/command_line_tools.rst

```diff
@@ -8,43 +8,47 @@ implement their functionality.
 Master program (crds)
 ---------------------
 
-Much like git,  the *crds* master command provides a wrapper around
-CRDS sub-commands like *list*, *certify*, *checksum*, *uniqname*, etc.::
+Much like git,  the `crds` master command provides a wrapper around
+CRDS sub-commands like `list`, `certify`, `checksum`, `uniqname`, etc.:
 
-   % crds --help
-   
-   usage: crds <command> <parameters...>
+  .. code-block:: bash
 
-  'crds' is a wrapper used to invoke various CRDS sub-programs.
+      $ crds --help
+   
+      usage: crds <command> <parameters...>
 
-   For example,  to assign HST best reference files,  do:
+`crds` is a wrapper used to invoke various CRDS sub-programs.
 
-     $ crds bestrefs --files *.fits  --update-bestrefs
+For example,  to assign HST best reference files,  do:
 
-   Available commands:
+  .. code-block:: bash
 
-   list                -- print information about CRDS configuration, etc. 
-   certify             -- check CRDS reference and rules files
-   bestrefs            -- assign bestrefs to datasets, regressions, repro
-   sync                -- manage local CRDS cache, download rules + references
-   diff                -- difference CRDS rules and references
-   rowdiff             -- difference reference tables
-   matches             -- list matching criteria relative to particular rules
-   checksum            -- update rmap checksum
-   query_affected      -- download CRDS new reference files affected dataset IDs
-   uniqname            -- rename HST files with new CDBS-style names
-   get_synphot         -- download synphot references
-   newcontext          -- automatically generates updated imaps and pmaps
-   refactor            -- automatically modifies rmaps
+      $ crds bestrefs --files *.fits  --update-bestrefs
 
-   For more detail about individual commands use --help:
+      Available commands:
```

Comment:
the `crds --help` command actually does print all of that - I mistakenly separated out part of the explanation thinking it wasn't supposed to be included, but turns out it all gets printed from the help command.  Will out it back to how it is in the original version.

Reply:
Unrelated: there are a number of places in this file where "--somearg" gets changed to a single dash "-somearg" so I'm going back through and adding "``" code wraps to prevent confusion. Will review the other files for this as well.